tests/DynamoDbClientServiceTest.php

```diff
@@ -13,7 +13,7 @@
  */
 class DynamoDbClientServiceTest extends TestCase
 {
-    public function setUp()
+    public function setUp(): void
```

Comment:
Are the type returns necessary? Would it be possible to remove these and drop the explicit php 7.1.3 requirement in `composer.json`? That could get us closer to the broadest range of Laravel compatibility.

Reply:
They are required by PHPUnit 8

Reply:
For development, I'm ok with the requirement of php7.1 and above.